ansible/roles/ci/code_coverage/cpp/finalize/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Playbook for finalizing the cpp code coverage module
+
+- name: Run gcovr
+  shell: bash -c "gcovr -r {{ros_workspace}} --xml-pretty > {{code_coverage_results_dir}}/coverage.xml"
+    chdir={{code_coverage_results_dir}}
+  tags:
```

Comment:
remove tags. They are no needed here. You have defined them in the docker_site.yml